### PR TITLE
Cleaned up migration writer tests

### PR DESCRIPTION
**Pull request by MarkusH**

### CODE REVIEW

File: tests/migrations/test_writer.py [hunk 1]
```diff
@@ -136,43 +136,49 @@ def assertSerializedFieldEqual(self, value):
...
+    def test_serialize_collections(self):
         self.assertSerializedEqual({1: 2})
         self.assertSerializedEqual(["a", 2, True, None])
         self.assertSerializedEqual({2, 3, "eighty"})
         self.assertSerializedEqual({"lalalala": ["yeah", "no", "maybe"]})
         self.assertSerializedEqual(_('Hello'))
-        # Builtins
+
+    def test_serialize_builtin_types(self):
         self.assertSerializedEqual([list, tuple, dict, set])
```
Comment (berkerpeksag) [inline tests/migrations/test_writer.py]: `frozenset` is missing.

Comment (MarkusH) [inline tests/migrations/test_writer.py]: Thanks.

**Comment by timgraham** [comment]
ship it.

**Comment by MarkusH** [comment]
Merged in 65d55c409343aab7c2ae771c459720ef797b4cdb

**Pull request closed without merging**